README.md

```diff
@@ -1,223 +1,107 @@
 
-## Setup
+# Key Features
 
-1. `cd utils/config-gen`
-2. `npm install`
-3. `npm start`
+- **Segment API Compatible**: RudderStack is Segment API and library compatible. So you don't need to change your app if you are using Segment.
 
-RudderStack config generator starts on the default port i.e., http://localhost:3000.
-On a successful setup, you should see the following
+- **Processing Flexibility**: With RudderStack's powerful JavaScript-based event transformation framework, you can enhance or transform your event data by combining it with your other internal data. Furthermore, since RudderStack runs inside your cloud or on-premise environment, you can easily access your production data to join with the event data.
 
-![image](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lq586FOQtfjJPKbd01W%2F-M0LidOVklHOkLYEulS4%2F-M0M3fzyrb-UHiNBG7j0%2FScreenshot%202020-02-18%20at%2012.20.57%20PM.png?alt=media&token=a3f24ad8-fe72-4fed-8953-8e4c790f6cfd)
+- **Unlimited Events**: Event volume-based pricing of most of the commercial systems is broken. With RudderStack, you are be able to collect as much data as possible without worrying about overrunning event budgets.
 
-## Export workspace config
+- **Stand-alone System**: RudderStack runs as a single Go binary with the only dependency being on the PostgreSQL database. There is no other dependency required to use RudderStack.
 
-After adding the required sources and destinations, export your workspace config. This workspace-config is required by the RudderStack Server.
-To learn more about adding sources and destinations in RudderStack, refer [Adding a Source and Destination in RudderStack](https://docs.rudderstack.com/getting-started/adding-source-and-destination-rudderstack)
+- **High Performance**: On a single m4.2xlarge, RudderStack can process 3000 events/second.
 
-Update the [config](https://docs.rudderstack.com/administrators-guide/config-parameters) variables `configFromFile` and `configJSONPath` in rudder-server to read workspace config from the exported JSON file.
+- **Enhanced Telemetry**: To help us improve RudderStack, we collect performance and diagnostic metrics about how you use RudderStack, and how it is working. **No customer data is present in the metrics**. For technical details, please check out our wiki page on [Telemetry](https://github.com/rudderlabs/rudder-server/wiki/Telemetry).
 
-## Start RudderStack with the workspace config file
 
-- Download the workspace config file on your machine.
-- In `docker-compose.yml`, uncomment `volumes` section under `backend` service. Specify the path to your workspace config.
-- In `build/docker.env`, set the environment variable `RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE=true`
+# RudderStack Architecture
 
----
+RudderStack has two major components, namely:
+- The Control Plane: Allows you to manage the configuration of your sources and destinations
```

Comment:
Can you add some kind of styling to emphasize on the words `Control Plane` and `Data Plane`

Reply:
Config-generator should be introduced here. We should talk about it before the RudderStack Setup. 

Two options for setting up the connections:
1. Managed control plane
2. Open source Config generator